sdk/helper/ldaputil/connection.go

```diff
@@ -14,7 +14,7 @@ import (
 // but through an interface.
 type Connection interface {
 	Bind(username, password string) error
-	Close()
+	Close() error
```

Comment:
I believe this was changed as a result of updating the `go-ldap/ldap` library to `v3.4.5` when `cap/ldap` was installed.